WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -24,15 +24,18 @@ public final class WordPressComOAuthClient: NSObject {
     @objc public static let WordPressComOAuthErrorNewNonceKey = "WordPressComOAuthErrorNewNonceKey"
     @objc public static let WordPressComOAuthErrorDomain = "WordPressComOAuthError"
 
+    @objc public static let WordPressComOAuthDefaultBaseUrl = "https://wordpress.com"
+    @objc public static let WordPressComOAuthDefaultApiBaseUrl = "https://public-api.wordpress.com"
+
     enum WordPressComURL: String {
-        case oAuthBase = "https://public-api.wordpress.com/oauth2/token"
-        case socialLogin = "https://wordpress.com/wp-login.php?action=social-login-endpoint&version=1.0"
-        case socialLogin2FA = "https://wordpress.com/wp-login.php?action=two-step-authentication-endpoint&version=1.0"
-        case socialLoginNewSMS2FA = "https://wordpress.com/wp-login.php?action=send-sms-code-endpoint"
-        case oAuthRedirect = "https://wordpress.com/"
-
-        var url: URL {
-            return URL(string: self.rawValue)!
+        case oAuthBase = "/oauth2/token"
+        case socialLogin = "/wp-login.php?action=social-login-endpoint&version=1.0"
+        case socialLogin2FA = "/wp-login.php?action=two-step-authentication-endpoint&version=1.0"
+        case socialLoginNewSMS2FA = "/wp-login.php?action=send-sms-code-endpoint"
+        case oAuthRedirect = "/"
```

Comment:
where are we using this? Couldn't find anything in the code

Reply:
Good point! I hadn't spotted that this wasn't used.